app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -24,4 +24,30 @@ body {
   margin-top: 0;
   margin-bottom: 0;
  }
+ .alert-error {
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
```

Comment:
several of these colors are duplicated - probably makes sense to abstract all of them into their own variables (makes this more readable, too). I.e.:
```
$light-pink: #f2dede;
... etc ...